doc/source/cookbook/thin_slice_projection.py

```diff
@@ -9,12 +9,12 @@
 # as a data_source for the projection.
 
 # Center on the domain center
-center = ds.domain_center
```

Comment:
it occurs to me that these arrays seem pretty cheap to copy. Would there be a clear downside to make the `MutableAttribute` descriptor always return copies ?

Reply:
Yeah, that's part of what we were trying to avoid with this.  They might be cheap, but if they get called an enormous number of times, they get expensive -- since we then also utilize a fair bit of unyt code.  That's what the original mutable attribute code did, to ensure that the underlying values were never changed.

Reply:
fair enough